package.json

```diff
@@ -241,7 +241,6 @@
     "get-port": "^5.0.0",
     "getopts": "^2.2.5",
     "getos": "^3.1.0",
-    "git-url-parse": "11.1.2",
```

Comment:
Nice catch!

We should remove the corresponding `@types/git-url-parse` as well